Explorer/Assets/DCL/ApplicationBlocklistGuard/ApplicationBlocklistGuard.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL.Diagnostics;
+using DCL.Multiplayer.Connections.DecentralandUrls;
+using DCL.WebRequests;
+using SceneRuntime.Apis.Modules.SignedFetch.Messages;
+using System.Linq;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.ApplicationBlocklistGuard
+{
+    public static class ApplicationBlocklistGuard
+    {
+        public static async UniTask<bool> CheckIfBlocklistedAsync(IWebRequestController webRequestController, IDecentralandUrlsSource urlsSource, string userID, CancellationToken ct)
+        {
+            FlatFetchResponse response = await webRequestController.GetAsync<FlatFetchResponse<GenericGetRequest>, FlatFetchResponse>(
+                urlsSource.Url(DecentralandUrl.Blocklist),
+                new FlatFetchResponse<GenericGetRequest>(),
+                ct,
+                ReportCategory.STARTUP,
+                new WebRequestHeadersInfo());
+
+            BlocklistData bd = JsonUtility.FromJson<BlocklistData>(response.body);
+
+            return bd.users.Select(u => u.wallet).Contains(userID);
```

Comment:
Optional: remove LINQ to reduce allocations. But not critical, since it happens only once at the startup time

Reply:
Hmm well for a once-off thing like this I'm not really worried about allocations, but I'll change this to .Any instead, which will only capture the userID, and is still cleaner than a foreach or something. We're probably overthinking it but why not 😄

Reply:
as I said, it is very optional, so we can keep it like it is.